### PR TITLE
Skipped disk activation step in case of installer updated (bsc#974409)

### PR DESCRIPTION
**Pull request by teclator**
During the tests i realized that the license agreement was not remembered so i also fix it.

### CODE REVIEW

File: src/lib/installation/clients/inst_disks_activate.rb [hunk 1]
```diff
@@ -55,6 +58,10 @@ def main
       @have_zfcp = false
       @want_fcoe = false
 
+      @installer_update ||= InstUpdateInstaller.new
```
Comment (ancorgs) [inline src/lib/installation/clients/inst_disks_activate.rb]: Weird use of a client. Is strange to have a client (something that is usually only meant to be executed) with the responsibility of knowing if the installer is updated.

Comment (ancorgs) [inline src/lib/installation/clients/inst_disks_activate.rb]: A better solution discussed offline

Comment (teclator) [inline src/lib/installation/clients/inst_disks_activate.rb]: Nice discussion ;)

**Comment by teclator** [comment]
This PR's are related to this one

yast/yast-yast2#463
yast/yast-packager#155

File: src/lib/installation/clients/inst_disks_activate.rb [hunk 2]
```diff
@@ -26,6 +26,9 @@
 #
 # $Id$
 #
+
+require "installation/clients/inst_update_installer"
```
Comment (ancorgs) [inline src/lib/installation/clients/inst_disks_activate.rb]: Is this needed?

**Comment by ancorgs** [comment]
Besides the surplus require, looks good.

**Pull request merged**